documentation/website/src/handbook/integration/serenity-2-migration-guide.md

```diff
@@ -0,0 +1,688 @@
+---
+title: Serenity/JS 2.x Migration Guide
+layout: handbook.hbs
+---
+
+# Serenity/JS 2.x Migration Guide
+
+## Introduction
+
+This guide is designed for people that have built Serenity/JS 1.x projects and wish to upgrade to Serenity/JS 2.x. The upgrade path is relatively straightforward as the main classes (`Actor`, `Task`, `Interaction` and `Question`) 
```

Comment:
Could we link `Actor`, `Task`, `Interaction`, `Question` and other classes we're referring to to their API docs?

Reply:
👍 done